packages/components/src/styles/components/side-nav/header.scss

```diff
@@ -69,21 +69,13 @@
 }
 
 // Dropdown & Button color theming overrides
-.hds-side-nav__dropdown, // deprecated classname
-.hds-side-nav,
-.hds-side-nav-header {
+
+.hds-side-nav {
   .hds-button,
-  .hds-dropdown-toggle-button,
-  .hds-dropdown-toggle-icon {
+  button[class*="hds-dropdown-toggle"] {
     // Apply dark theme to child interactive components not within a nested dropdown
-    &:not(
-      .hds-dropdown * *,
-      .ember-basic-dropdown-trigger * *,
-      .ember-basic-dropdown-content * *,
-      .hds-dialog-primitive__wrapper *
-    ) {
+    &:not(.hds-dropdown__content *, .ember-basic-dropdown-content *, .hds-dialog-primitive__wrapper *) {
```

Comment:
I have struggled so much reasoning with this composed/nested CSS selector. 
What if we change the way in which we declare it, so that is more streamlined, explicit, and easier to grasp (at least for me)?

```
.hds-app-header {
  :not(.hds-dropdown__content, .ember-basic-dropdown-content, .hds-dialog-primitive__wrapper) {
    .hds-button,
    .hds-dropdown-toggle-button,
    .hds-dropdown-toggle-icon {
      ...
    }
  }
}
```